src/index.tsx

```diff
@@ -80,8 +81,14 @@ export function Playground({
   code,
   height = '200px',
   id,
+  wrappingComponent: WrappingComponent,
   ...editorProps
-}: Partial<StoryState> & { height?: string; id?: string }) {
+}: Partial<StoryState> & {
+  height?: string;
+  id?: string;
+  wrappingComponent?: React.JSXElementConstructor<{ children: React.ReactNode }>;
```

Comment:
There is also `React.ComponentType` but it's not much different from `JSXElementConstructor`.